plugins/optimization-detective/class-od-link-collection.php

```diff
@@ -139,15 +177,22 @@ static function ( array $carry, array $link ): array {
 		// Add media attributes to the deduplicated links.
 		return array_map(
 			static function ( array $link ): array {
-				$media_attributes = array( 'screen' );
```

Comment:
The `screen` media type is no longer considered the default because arbitrary links can now be added, not just preloads. And consider for Embed Optimizer it may be that (hypothetically) an audio-only browser could load a page which includes a YouTube embed. This embed should still get preconnected even if there isn't anything visual on the screen. Granted, this is highly hypothetical 😄